ThirdParty/dawn/CMakeLists.txt

```diff
@@ -74,6 +74,49 @@ set(DAWN_ABSEIL_DIR         "${abseil-cpp_SOURCE_DIR}"     CACHE STRING "Directo
 
 FetchContent_MakeAvailable(dawn)
 
+if (PLATFORM_EMSCRIPTEN)
+    set(TARGETS
+        tint_lang_core 
+        tint_lang_core_type
+        tint_lang_core_constant
+        tint_lang_core_ir
+        tint_lang_core_ir_transform
+        tint_lang_core_ir_transform_common 
+        tint_lang_core_intrinsic
+        tint_lang_spirv_validate
+        tint_lang_spirv_reader_parser
+        tint_lang_spirv_reader_lower
+        tint_lang_spirv_reader_ast_parser
+        tint_lang_spirv_reader_ast_lower
+        tint_lang_wgsl_ir 
+        tint_lang_wgsl_ast
+        tint_lang_wgsl_sem
+        tint_lang_wgsl_program
+        tint_lang_wgsl_resolver
+        tint_lang_wgsl_reader
+        tint_lang_wgsl_reader_parser
+        tint_lang_wgsl_reader_lower
+        tint_lang_wgsl_intrinsic
+        tint_lang_wgsl_inspector
+        tint_lang_wgsl_ast_transform 
+        tint_lang_wgsl_writer_ast_printer
+        tint_lang_wgsl_writer_raise 
+        tint_lang_wgsl_writer_ir_to_program
+        tint_lang_wgsl_reader_program_to_ir
+        tint_utils_id
+        tint_utils_text
+        tint_utils_symbol
+        tint_utils_diagnostic
+        tint_utils_result
+    )
```

Comment:
We have a helper CMake function [`find_targets_in_directory`](https://github.com/DiligentGraphics/DiligentCore/blob/master/BuildTools/CMake/BuildUtils.cmake#L207)